pages/windows/curl.md

```diff
@@ -8,7 +8,11 @@
 
 - View documentation for the original `curl` command:
 
-`tldr curl -p common`
+`tldr curl --platform common`
+
+- View documentation for the original `curl` command in older versions of `tldr` command-line client:
+
+`tldr curl --os common`
```

Comment:
I have no ideas whether reject or accept it so I'll accept it.

Reply:
It's slightly different than common alias pages as this command requires the `--platform` parameter to be passed.

+ `tldr invoke-webrequest` won't be affected in Node client and other, spec-compliant clients,
+ But `tldr curl --platform common` will be affected as `--platform` isn't accepted as a param in the Node client